MonkeyChat/ChatViewController.swift

```diff
@@ -28,7 +29,13 @@ class ChatViewController: JSQMessagesViewController, JSQMessagesComposerTextView
   
   let maxSize = 8500
   
+  // DATA - conversation
   var conversation: MOKConversation!
+  var memberHash = [String:MOKUser]() // monkeyId: MOKUser
+  var nameMembers = [String]()
+  var nameMembersDescription: String?
```

Comment:
let's erase `nameMembersDescription` and get its value by calling `self.nameMembers.joined(separator: ", ")`